build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__/SalesforceSDKAssets.xcassets/login-window-gear.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
As @wmathurin mentioned, we probably want these to be symlinks to a common shared folder like ```Settings.bundle``` is/was.